server/src/library/libraryUtils.ts

```diff
@@ -9,13 +9,17 @@ import * as StandardLibrarySymbolsEnUs from "./standard/standard-enUs.json";
 import { createLibraryTypeResolver, LibraryDefinitionsGetter } from "./libraryTypeResolver";
 import { PartialResult, PartialResultUtils } from "@microsoft/powerquery-parser";
 
-export function getOrCreateStandardLibrary(locale?: string): Library.ILibrary {
+export function getOrCreateStandardLibrary(
+    locale?: string,
+    otherLibraryDefinitionsGetters: LibraryDefinitionsGetter[] = [],
```

Comment:
I don't feel strongly about it, but my preference would actually be towards using optional parameters / default values. Passing in an empty array parameter just feels weird, and I'd want to see a code comment as to why it is there. like

```typescript
getOrCreateStandardLibrary(currentLocale, /*other library definitions*/ [])
```
I'm fine with removing the optional parameters, but I'd suggest we do that as a separate change. Ideally it would be enforceable with an eslint rule as well.